src/main/java/org/alfresco/rest/api/impl/NodesImpl.java

```diff
@@ -2313,6 +2313,7 @@ else if (! destNodeTypeQName.equals(nodeTypeQName))
 
             try
             {
+                handleUpdateNodeName(props, nodeRef);
```

Comment:
Can you please change the name of the method to "handleNodeRename", I think it is more descriptive.